doc/manual/types.rst

```diff
@@ -800,6 +800,20 @@ Constructor methods to appropriately handle such mixed cases can be
 defined, but that will not be discussed until later on in
 :ref:`man-constructors`.
 
+Due to type invariance, the following method does not accept arguments of type ``Point{Float64}``::
+
+    function norm(p::Point{Real})
+       sqrt(p.x^2 + p.y^2)
+    end
+
+The correct way to define a method that accepts all arguments of ``Point`` types where ``T`` is a subtype of ``Real`` is::
+
+    function norm{T <: Real}(p::Point{T})
```

Comment:
It looks like the convention is to write `{T<:Real}` to me (though it's hard to be sure...).